tuxemon/client.py

```diff
@@ -273,11 +273,8 @@ def _send_event(
         """
         for state in self.active_states:
             maybe_game_event = state.process_event(game_event)
-            if maybe_game_event is None:
-                break
-            game_event = maybe_game_event
-
-        return maybe_game_event
+            return maybe_game_event
```

Comment:
At this point, we can directly return the result of `process_event` call, no need to store it in a variable.
```suggestion
            return state.process_event(game_event)
```

Reply:
Btw why did you remove the break logic & the "bubble out" of the event to the next state logic?
Cf. my comments on Discord.

Reply:
@Grimmys I tried to see if without the break logic was working and I didn't notice any weird behavior, opening, closing, saving, loading, teleporting, no issue with any of these.

Reply:
fixed